### PR TITLE
Pantheon sites accounts rename

### PR DESCRIPTION
**Pull request by dalin-**
Script to change usernames and email addresses from old to new.

See attached .csv file

Run like
```
pantheon-sites-accounts-rename transition.csv
```
[transition.csv](https://github.com/fourkitchens/pantheon-tools/files/7087977/transition.csv)

### CODE REVIEW

**Comment by dalin-** [comment]
Oh, I should note that I haven't actually run the script yet on all the sites, only on a few sample sites in the dev environment only.  So if you do want to try and run it, you'll want to uncoment one of the lines that overwrites the `$sites` variable.

**Comment by heatherwoz** [comment]
This seems risky to run all at once. What if something goes wrong and you have to restart? What if you want some control over the timing of when it happens on a particular site? What if you don't want to run it against some sites? I think breaking the user account updates and the security notice updates into two separate scripts would be good. Or providing a way to specify a list of sites that you want to run it on.

File: pantheon-sites-accounts-rename [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+        else {
+          echo "  notice: no security update emails to convert." . PHP_EOL;
+        }
+
+        break;
+
+      case "drupal8":
+        // Security update notification.
+        if ($environment === 'dev') {
+          // We're going to write/deploy config later, so pull a fresh copy of
```
Comment (heatherwoz) [inline pantheon-sites-accounts-rename:134]: this makes me nervous. what if there is WIP or undeployed code on dev?

**Comment by dalin-** [comment]
@heatherwoz 

> This seems risky to run all at once. What if something goes wrong and you have to restart? What if you want some control over the timing of when it happens on a particular site? What if you don't want to run it against some sites?

Fair enough.  I've modified it to ask for a list of sites rather than run on everything. 

>  I think breaking the user account updates and the security notice updates into two separate scripts would be good.

I didn't do this part.  If on a particular site you want to change the security updates separately it's probably quicker to just do it by hand. 

> this makes me nervous. what if there is WIP or undeployed code on dev?

Far enough.  The script now gives you the option of pausing on each dev env. for you to deploy manually.

**Pull request merged**